src/main/java/cinema/service/MovieSessionService.java

```diff
@@ -0,0 +1,11 @@
+package cinema.service;
+
+import cinema.model.MovieSession;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public interface MovieSessionService {
+    MovieSession add(MovieSession movieSession);
+
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDateTime date);
```

Comment:
```suggestion
    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
```
давай реалізуємо згідно з умовою